src/components/containers/DashboardContainer.js

```diff
@@ -24,17 +24,15 @@ class DashboardContainer extends React.Component {
         if (dataFetch.pending) {
             return <Spinner />;
         } else if (dataFetch.rejected) {
-            const extraErrorComponentProps = {};
-
             if (dataFetch.reason && dataFetch.reason.message) {
-                extraErrorComponentProps.message = dataFetch.reason.message;
+                // eslint-disable-next-line no-console
+                console.error(dataFetch.reason.message);
```

Comment:
Do we have server logging for these? Like, do we know how common a dashboard-fetch-error is, or if it is a server issue?

Reply:
We don't, unfortunately. It's all static hosting.

Reply:
I suppose we could integrate with something like Sentry. Feel free to open an issue! :smiley: